x-pack/test/saved_object_api_integration/common/fixtures/saved_object_test_plugin/server/plugin.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PluginInitializerContext, CoreSetup, CoreStart, SavedObject } from 'kibana/server';
+
+export class Plugin {
+  constructor(initializerContext: PluginInitializerContext) {}
```

Comment:
__optional nit:__ seems `initializerContext` isn't used (same for spaces test plugin).